pkg/model/storage_spec.go

```diff
@@ -20,6 +20,9 @@ type StorageSpec struct {
 	// Source URL of the data
 	URL string `json:"URL,omitempty"`
 
+	// URL of the git Repo to clone
+	Repo string `json:"Repo,omitempty"`
+
```

Comment:
Why not use the URL field for this?

Reply:
I tried using the URL field for it but failed, but I think if we use the URL field it can't be easily distinguished whether an URL is an URL that is to be used to clone the repo